scripts/project.py

```diff
@@ -33,10 +35,10 @@ class Project(metaclass=abc.ABCMeta):
     """
     securify_jar = pathlib.Path("build/libs/securify-0.1.jar")
 
-    def __init__(self, project_root, pretty_output):
+    def __init__(self, project_root, clargs):
```

Comment:
What does clargs stand for, clang arguments?

Reply:
I stands for command line args. I'll change it to just `args` if it's unclear.